zebra/zebra_srte.c

```diff
@@ -81,6 +81,60 @@ struct zebra_sr_policy *zebra_sr_policy_find_by_name(char *name)
 	return NULL;
 }
 
+static int process_routes_for_policy(struct zebra_sr_policy *policy,
+				     struct zserv *client, uint32_t message,
+				     struct stream *s, struct zapi_nexthop *znh,
+				     unsigned long *nump)
+{
+	struct route_table *table;
+	struct route_node *rn;
+	struct route_entry *re, *next;
+	int num = 0;
+	int ret;
+
+	/* Lookup table.  */
+	table = zebra_vrf_table(AFI_IP6, SAFI_UNICAST, VRF_DEFAULT);
+	if (table == NULL)
+		return -1;
+
+	for (rn = route_top(table); rn; rn = route_next(rn)) {
+		RNODE_FOREACH_RE_SAFE (rn, re, next) {
+			if (re->nhe->nhg.nexthop->srte_color == policy->color &&
+			    IPV6_ADDR_CMP(&re->nhe->nhg.nexthop->gate,
+					  &policy->endpoint)) {
+				if (num == 0) {
+					stream_putc(s, ZEBRA_ROUTE_SRTE);
+					stream_putw(s,
+						    0); /* instance - not available */
+					stream_putc(s, 150); /* TODO: distance */
```

Comment:
replace 150 with ZEBRA_SHARP_DISTANCE_DEFAULT (defined in zebra.h)
if there is a problem with the given distance value, changes should be done in zebra.h, now.

Reply:
fixed